site/en/blog/extension-instantnav/index.md

```diff
@@ -0,0 +1,199 @@
+which is a unique identifier for each document. If a frame is navigated and
+opens a new document the identifier will change. This field is useful for
+determining when pages change their life cycle state (between
+prerender/active/cached) because it remains the same.
+
+### Web navigation events
+
+Events in the [`chrome.webNavigation` namespace] can fire multiple times on the
+same page depending on the life cycle it is in. See
+[“How do I tell what life cycle the page is in?”](#lifecyle)
+and [“How do I determine when a page transitions?”](#transitions) sections.
+
+## How do I tell what life cycle the page is in? {: #lifecyle}
+
+The [`DocumentLifecycle`](/docs/extensions/reference/extensionTypes/#type-DocumentLifecycle)
+type has been added to a number of extensions APIs where the `frameId` was
+previously available. If the `DocumentLifecycle` type is present on an event
+(such as [`onCommitted`](/docs/extensions/reference/webNavigation/#event-onCommitted)),
+its value is the state in which the event was generated. You can always query
+information from the `WebNavigation` [`getFrame()`](/docs/extensions/reference/webNavigation/#method-getFrame)
+and [`getAllFrames()`](/docs/extensions/reference/webNavigation/#method-getAllFrames)
+methods, but using the value from the event is always preferred. If you do use
+either method be aware the state of the frame may change between when the event
+was generated and when the promises return by both methods is resolved.
+
+The [`DocumentLifecycle`](/docs/extensions/reference/extensionTypes/#type-DocumentLifecycle)
+has the following values:
+
+- `"prerender`" : Not currently presented to the user but preparing to possibly be displayed to the user.
+- `"active"`: Presently displayed to the user.
+- `"cached"`: Stored in the Back/Forward Cache.
+- `"pending_deletion"`:  The document is being destroyed.
+
+## How do I determine if a frame is the outermost frame? {: #outermost-frame}
+
+Previously extensions may have checked whether `frameId == 0` to determine
+if the event occurring is for the outermost frame or not. With multiple pages
+in a tab we now have multiple outermost frames, so the definition of *frameId*
+is problematic. You will never receive events about a Back/Forward cached
+frame. However, for prerendered frames the `frameId` will be
+non-zero for the outermost frame. So using `frameId == 0` as a signal for
+determining if it is the outermost frame is incorrect.
+
+To help with this, we introduced a new type called [`FrameType`](/docs/extensions/reference/extensionTypes/#type-FrameType)
+so determining if the frame is indeed the outermost frame is now easy.
+
+[`FrameType`] has the following values:
+
+- `"outermost_frame"`: Typically referred to the topmost frame. Note that there are multiple of
+these: there could be many for the various documentLifecycles (such as a
+prerendered outermost_frame and a cached outermost_frame).
```

Comment:
```suggestion
- `"outermost_frame"`: Typically referred to the topmost frame. Note that there are multiples of
these. For example, if you have prerendered and cached pages, each has an outermost frame that could be called its topmost frame.
```

Reply:
The first part of the description is missing a word. Which of these is correct:

* Typically refers to the topmost frame.
* Typically referred to as the topmost frame.

Reply:
Done